src/Common/CodeEditor/CodeEditor.tsx

```diff
@@ -1,108 +1,44 @@
-import React, { useEffect, useCallback, useReducer, useRef } from 'react'
+import React, { useEffect, useReducer, useRef } from 'react'
 import MonacoEditor, { MonacoDiffEditor } from 'react-monaco-editor'
-// import { useJsonYaml, Select, RadioGroup, useWindowSize, copyToClipboard } from '../common'
-// import { Progressing } from '@devtron-labs/devtron-fe-common-lib'
 import YAML from 'yaml'
+import ReactGA from 'react-ga4'
```

Comment:
Suggestion: we can remove the same from eslintignore

Reply:
Done